whylog_vim/output_formater/output_agregator.py

```diff
@@ -0,0 +1,34 @@
+from functools import partial
+from whylog_vim.consts import Messages
+
+
+class OutputAgregator(object):
+    def __init__(self):
+        self.buttons = {}
+        self.output_lines = []
+
+    def add(self, element):
+        self.output_lines.append(element)
+
+    def add_commented(self, content):
+        self.add(Messages.COMMENT_PREFIX % content)
+
+    def get_content(self):
+        # The same type as vim buffor.
+        return self.output_lines
+
+    def create_button(self, click_function, **params):
+        line = self._get_current_line_number()
+        self.buttons[line] = partial(click_function, **params)
```

Comment:
callback is a pretty standard thing and this is not. Please change the function interface so that it accepts a callback function - if the caller needs to apply some parameters, then *he* should be the one to use `functools.partial`.